addons/social_share/wizards/mail_compose.message.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class MailComposer(models.TransientModel):
+    """Override of mail composer to be able to send templates without specified recipients."""
+    _inherit = 'mail.compose.message'
+
+    force_default_recipients = fields.Boolean()
+
+    def _prepare_mail_values_dynamic(self, res_ids):
```

Comment:
This  #will have to be cleaned somehow :) don't think we should hack the values generation like that. Probably ``action_send`` should somehow force partner_ids when instantiating the composer (especially if we get rid of templates, it will be easier to force the values of the composer).